### ISSUE TITLE
Two questions about hand keypoints detection

### ISSUE THREAD

**Issue by zy1996829** [facebookresearch/InterHand2.6M#140]
1.During actual testing, I found that the depth information of the hand key points output by real-time testing did not show any regular changes when the hand was close to or far from the camera.Does the third dimension in the final output of the hand keypoint coordinates represent the depth value of the hand keypoints in the real world? If I need to detect the movement of hands in the z-axis direction, can I use the third dimension in the coordinates output by the model to determine it?

2.I conducted real-time keypoint detection of human hands using a camera, but found that the detection accuracy of keypoints is higher when the hands are closer to the camera, while the detection accuracy becomes lower and chaotic when the hands are slightly further away from the camera. What could be causing this issue? As shown in the figure below:
![image1](https://github.com/facebookresearch/InterHand2.6M/assets/26729385/c61ca399-a835-4fa5-ab9c-f64a1eb66e5c)

![image2](https://github.com/facebookresearch/InterHand2.6M/assets/26729385/7cf80b80-efa3-4d9c-83b5-85baf6931b8c)
![image3](https://github.com/facebookresearch/InterHand2.6M/assets/26729385/a01f2c65-6263-448b-894c-62ce078a4a6d)
![image4](https://github.com/facebookresearch/InterHand2.6M/assets/26729385/af1b5dc1-17c6-4eca-adbc-c87aa8caf5a1)
![image5](https://github.com/facebookresearch/InterHand2.6M/assets/26729385/3a01aae1-2dc4-4580-885e-15c63f5dc31c)

**Comment by mks0601** [comment]
Hi, that is expected as InterHand2.6M consists of Lab images whose appearances is far from in-the-wild images. Maybe you should try this, which works robust on in-the-wild images: https://github.com/facebookresearch/InterWild

**Comment by zy1996829** [comment]
> Hi, that is expected as InterHand2.6M consists of Lab images whose appearances is far from in-the-wild images. Maybe you should try this, which works robust on in-the-wild images: https://github.com/facebookresearch/InterWild

Ok, I will check it. Another question I have now is that I trained a new model using a subset of InterHand dataset,with 20 epochs,but the performance is much worse than your pre-trained model,as the key points cannot be detected.

**Comment by mks0601** [comment]
I'm not sure what you mean by the performance, but that is a trade-off between efficiency and effectiveness.
